Meta/CMake/flatpak/org.ladybird.Ladybird.json

```diff
@@ -0,0 +1,498 @@
+          "-DJPEGXL_BUNDLE_LIBPNG=OFF",
+          "-DCMAKE_INSTALL_RPATH=\"$ORIGIN:$ORIGIN/../lib64:/app/lib64\""
+        ]
+    },
+    {
+      "name": "skia",
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://skia.googlesource.com/skia.git",
+          "commit": "chrome/m129"
+        },
+        {
+          "type": "file",
+          "path": "skia-configure.sh"
+        },
+        {
+          "type": "file",
+          "path": "skia-build.sh"
+        },
+        {
+          "type": "file",
+          "path": "skia-install.sh"
+        }
+      ],
+      "build-commands": [
+        "./skia-configure.sh",
+        "./skia-build.sh",
+        "./skia-install.sh"
+      ]
+    },
+    {
+      "name": "libtommath",
+      "buildsystem": "cmake-ninja",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/libtom/libtommath.git",
+          "tag": "v1.3.0"
+        }
+      ],
+      "config-opts": [
+        "-DCMAKE_BUILD_TYPE=Release",
+        "-DCMAKE_PREFIX_PATH=/app",
+        "-DCMAKE_INSTALL_LIBDIR=lib",
+        "-DBUILD_SHARED_LIBS=ON"
+      ]
+    },
+    {
+      "name": "fast_float",
+        "buildsystem": "cmake-ninja",
+        "sources": [
+          {
+            "type": "git",
+            "url": "https://github.com/fastfloat/fast_float.git",
+            "tag": "v8.0.2"
+          }
+        ],
+        "config-opts": [
+          "-DCMAKE_BUILD_TYPE=Release",
+          "-DCMAKE_PREFIX_PATH=/app",
+          "-DFASTFLOAT_INSTALL=ON"
+        ]
+    },
+    {
+      "name": "Ladybird",
+      "buildsystem": "cmake-ninja",
+      "sources": [
+        {
+          "type": "file",
+          "url": "https://curl.se/ca/cacert-2025-05-20.pem",
+          "dest": "Caches/CACERT/",
+          "sha256": "ab3ee3651977a4178a702b0b828a4ee7b2bbb9127235b0ab740e2e15974bf5db"
+        },
+        {
+          "type": "file",
+          "url": "https://raw.githubusercontent.com/publicsuffix/list/32c68ce9d52e12df5b914b2b248cb893147301f7/public_suffix_list.dat",
+          "dest": "Caches/PublicSuffix/",
+          "sha256": "e79e372bcc6fcdb51f7a31e3c0c504530838432669af2ac544d2491de0a86030"
+        },
+        {
+          "type": "shell",
+          "commands": [
+            "echo -n 2025-05-20 > Caches/CACERT/version.txt",
+            "# FIXME Is there an easier way to set an absolute path in config-opts?",
+            "echo set\\(LADYBIRD_CACHE_DIR \"$PWD/Caches\" CACHE STRING \\\"\\\"\\) > Cache.cmake"
+          ]
+        },
+        {
+          "type": "dir",
+          "path": "../../../"
+        }
+      ],
+      "config-opts": [
+        "-CCache.cmake",
+        "-DCMAKE_PREFIX_PATH=/app",
+        "-DCMAKE_INSTALL_LIBDIR=lib",
+        "-DCMAKE_BUILD_TYPE=Release",
+        "-DCMAKE_C_COMPILER=clang",
+        "-DCMAKE_CXX_COMPILER=clang++",
+        "-DLAGOM_USE_LINKER=lld",
```

Comment:
clang not required. not yet anyway. can probably remove and keep things more gcc-like